pkgs/by-name/cb/cbmp/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  buildNpmPackage,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildNpmPackage rec {
```

Comment:
Can you try using `yarn{Config,Build}Hook` instead of buildNpmPackage?

Reply:
I don't think this applies here.

Reply:
Why not? There is a `yarn.lock` in the repository.